backend/src/instagram/instagram.service.ts

```diff
@@ -30,7 +30,6 @@ export class InstagramFeedService {
     async fetchAndSaveYouTubeFeed() {
         this.logger.log('Fetching Instagram feeds Started');
         await instagram.get('users/self/media/recent')
-            .then(this.filterAlreadyExistingFeeds)
             .then(this.formatFeed)
             .then(this.persistIntoDb)
```

Comment:
This method needs to be modified since this is saving an array of entities. Since you have added unique constraint if either one those unique constraint fails all of the save will fail. This is applicable to all the service classes.